01_fib/kasunfdo/main.go

```diff
@@ -0,0 +1,36 @@
+package main
```

Comment:
This file doesn't belong in this PR.

Reply:
It wound up in the main repo somehow. I'll remove it.